migrations/20170206131527_users.js

```diff
@@ -1,7 +1,7 @@
 exports.up = (knex) => {
   return knex.schema.createTable('users', (table) => {
     table.increments();
-    table.integer('linkedin_id').notNullable().unique();
+    table.string('linkedin_id').notNullable().unique();
```

Comment:
The Id that LinkedIn returns is a hash-like string, it is not an integer and cannot be stored as one.